### ISSUE TITLE
If you try to use system shortcuts while on full screen, the UI get's broken

### ISSUE THREAD

**Issue by csm2k3** [pop-os/cosmic-epoch#646]
In Firefox, and other browsers, if you put it in full screen and try to use a shortcut e.g opening the launcher or doing alt+tab, two blank bars appear on the top and bottom of the application (it's probably the top panel and the dock) and the application turns unresponsive (browsers).

Tested on Visual Studio Code, and while the UI still gets broken the application itself (code) works normally.

Edit: the correct term is OSD

**Comment by vrihaspati110** [comment]
I was able to reproduce it as well. Getting out of full-screen fixed the issue.

**Comment by HafizBeyy** [comment]
yeah, dock and title bar disappears as a blank box . also cannot alt tab to other open windows. _**My sistem is:**_ 
![screenshot-2024-08-13-13-40-27](https://github.com/user-attachments/assets/e9bc2cec-94d4-459c-b0f6-3d691af06358)

**Comment by HafizBeyy** [comment]
also a showing popup disables fullscreen. 
![screenshot-2024-08-13-13-44-04](https://github.com/user-attachments/assets/63e579c7-eb11-42fd-be07-b4052c3d6edc)
it was fullscreen before i showed open folder shortcut via key shortcut

**Comment by brunizzl** [comment]
While this is not a big issue for most shortcuts, as most are pressed when some focus outside the current fullscreen application is wanted, there is one specific case i find quite annoying: the keyboard media control keys. Especially volume adjustment becomes tedious.

**Comment by KodeBarista** [comment]
I think this issue has been fixed. I tested with Zen (Firefox-based) and VS Code, and both are working without issues.
